bitnami/argo-cd/values.yaml

```diff
@@ -0,0 +1,1991 @@
+  ## @param server.autoscaling.enabled Enable Argo CD server deployment autoscaling
+  ## @param server.autoscaling.minReplicas Argo CD server deployment autoscaling minimum number of replicas
+  ## @param server.autoscaling.maxReplicas Argo CD server deployment autoscaling maximum number of replicas
+  ## @param server.autoscaling.targetCPU Argo CD server deployment autoscaling target CPU percentage
+  ## @param server.autoscaling.targetMemory Argo CD server deployment autoscaling target CPU memory
+  ##
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 5
+    targetCPU: 50
+    targetMemory: 50
+
+  ## Redirect all request to https
+  ## @param server.insecure Disable HTTPS redirection for Argo CD server
+  ##
+  insecure: false
+
+  ## @param server.logFormat ArgoCD server logs format. Options: [text, json]
+  ##
+  logFormat: text
+  ## @param server.logLevel ArgoCD server logs level
+  ##
+  logLevel: info
+
+  ## Argo CD server enable config
+  ## @param server.configEnabled Enable Argo CD server config
+  ## Ref: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cm.yaml
+  ##
+  configEnabled: true
+
+  ## Argo CD server config. This object will be directly rendered
+  ## Ref: https://argoproj.github.io/argo-cd/operator-manual/user-management/
+  ## E.g:
+  ## repositories: |
+  ##   - url: git@github.com:group/repo.git
+  ##     sshPrivateKeySecret:
+  ##       name: secret-name
+  ##       key: sshPrivateKey
+  ##   - type: helm
+  ##     url: https://charts.helm.sh/stable
+  ##     name: stable
+  ##   - type: helm
+  ##     url: https://argoproj.github.io/argo-helm
+  ##     name: argo
+  ## oidc.config: |
+  ##   name: AzureAD
+  ##   issuer: https://login.microsoftonline.com/TENANT_ID/v2.0
+  ##   clientID: CLIENT_ID
+  ##   clientSecret: $oidc.azuread.clientSecret
+  ##   requestedIDTokenClaims:
+  ##     groups:
+  ##       essential: true
+  ##   requestedScopes:
+  ##     - openid
+  ##     - profile
+  ##     - email
+  ## dex.config: |
+  ##    connectors:
+  ##      # GitHub example
+  ##      - type: github
+  ##        id: github
+  ##        name: GitHub
+  ##        config:
+  ##          clientID: aabbccddeeff00112233
+  ##          clientSecret: $dex.github.clientSecret
+  ##          orgs:
+  ##          - name: your-github-org
+  config:
+    ## Argo CD external base URL. Required when configuring SSO. Required when enabling dex.
+    ## @param server.config.url Argo CD server url. Required when enabling dex.
+    ## E.g:
+    ## url: https://argocd.example.com
+    ##
+    url: ""
+    ## Argo CD instance label key
+    ## @param server.config.application.instanceLabelKey
+    ##
+    application.instanceLabelKey: argocd.argoproj.io/instance
+    ## If Dex is enabled you need to add connectors here
+    ## @param server.config.dex.config Dex config that will end in the argocd-cm
+    ## dex.config: |
+    ##  connectors: []
+    ##
+    dex.config: ""
```

Comment:
If it is a value in the chart, then I'd use the CamelCase naming that we use in the rest of the charts

Reply:
Maybe I did not clarify it enough. This is a value that will be rendered as the key in the configmap, it is required by Argo CD for example that the CM key is called `dex.config`, so it is used to be rendered in the configmap (`server.config` is not a string field) and we cannot use camelCase because that's the required key name (including the `.`)